lib/ansible/executor/task_executor.py

```diff
@@ -395,8 +393,6 @@ def _execute(self, variables=None):
         if variables is None:
             variables = self._job_vars
 
-        templar = Templar(loader=self._loader, shared_loader_obj=self._shared_loader_obj, variables=variables)
```

Comment:
This may not be backwards compatible.  It looks like here we can pass in `variables`.  If `variables` is passed into `_execute` then `templar` doesn't have those explicitly passed variables.

Although not sure that is really an issue.  Looks like we don't call `_execute` anywhere with `variables`.